.github/workflows/rector.yaml

```diff
@@ -64,17 +64,13 @@ jobs:
 
             -   run: vendor/bin/ecs check --fix --ansi
 
-            # see https://github.com/EndBug/add-and-commit
+            # see https://github.com/rectorphp/rector/issues/7579
             -
                 # commit only to core contributors who have repository access
                 if: github.event.pull_request.head.repo.full_name == github.repository
-                uses: EndBug/add-and-commit@v7.5.0
```

Comment:
Would new https://github.com/EndBug/add-and-commit 9.1 help?

Reply:
that not work, see dependabot PR: https://github.com/rectorphp/rector-src/pull/3026